app/delivery/send_to_providers.py

```diff
@@ -26,7 +26,7 @@
     SMS_TYPE,
     KEY_TYPE_TEST,
     BRANDING_BOTH,
-    BRANDING_ORG_BANNER,
+    BRANDING_ORG_BANNER_NEW,
```

Comment:
Let's add a comment here to outline why this was needed.  

Can link right to the line of the other file.

As a new dev to the project seeing `BRANDING_ORG_BANNER_NEW` would leave a pick question mark as to what is going on.